src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -814,28 +814,18 @@ namespace ZoneAirLoopEquipmentManager {
             }
         }
         if (ProvideSysOutput) {
+            int OutletNodeNum = AirDistUnit(AirDistUnitNum).OutletNodeNum;
+            int ZoneAirNode = ZoneEquipConfig(ControlledZoneNum).ZoneNode;
+            SpecHumOut = Node(OutletNodeNum).HumRat;
+            SpecHumIn = Node(ZoneAirNode).HumRat;
             // Sign convention: SysOutputProvided <0 Zone is cooled
             //                  SysOutputProvided >0 Zone is heated
-            SpecHumOut = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).HumRat;
-            SpecHumIn = Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat;
-            if (AirDistUnit(AirDistUnitNum).EquipType_Num(1) == SingleDuctConstVolNoReheat) {
-                // Use old direct air method to avoid diffs for now
-                SysOutputProvided = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).MassFlowRate *
-                                    (Psychrometrics::PsyHFnTdbW(Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).Temp,
-                                                                Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat) -
-                                     Psychrometrics::PsyHFnTdbW(Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).Temp,
-                                                                Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat));
-            } else {
-                Real64 CpAirAvg = PsyCpAirFnW(0.5 * (SpecHumOut + SpecHumOut));
-                SysOutputProvided = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).MassFlowRate * CpAirAvg *
-                                    (Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).Temp - Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).Temp);
-            }
-
+            Real64 CpAirAvg = PsyCpAirFnW(SpecHumOut);
+            SysOutputProvided = Node(OutletNodeNum).MassFlowRate * CpAirAvg * (Node(OutletNodeNum).Temp - Node(ZoneAirNode).Temp);
```

Comment:
The previous calculation for constant volume no reheat air terminal unit was using enthalpy difference with humidity ratio at the zone condition. Using zone humidity ratio is not realistic representation as it is lagged value. This fix uses air terminal outlet node humidity ratio and is consistent across all air terminal units.